Source/RuntimeMeshComponent/Private/RuntimeMeshComponentProxy.cpp

```diff
@@ -318,50 +318,50 @@ void FRuntimeMeshComponentSceneProxy::GetDynamicMeshElements(const TArray<const
+	//SCOPE_CYCLE_COUNTER(STAT_RuntimeMeshComponentSceneProxy_GetDynamicRayTracingInstances);
 
-	// TODO: Should this use any LOD determination logic? Or always use a specific LOD?
-	int32 LODIndex = 0;
-	
-	if (RuntimeMeshProxy->HasValidLODs())
-	{
-		auto& LOD = RuntimeMeshProxy->GetLOD(LODIndex);
+	//// TODO: Should this use any LOD determination logic? Or always use a specific LOD?
+	//int32 LODIndex = 0;
+	//
+	//if (RuntimeMeshProxy->HasValidLODs())
+	//{
+	//	auto& LOD = RuntimeMeshProxy->GetLOD(LODIndex);
 
-		for (const auto& SectionEntry : LOD.Sections)
-		{
-			int32 SectionId = SectionEntry.Key;
-			auto& Section = SectionEntry.Value;
+	//	for (const auto& SectionEntry : LOD.Sections)
+	//	{
+	//		int32 SectionId = SectionEntry.Key;
+	//		auto& Section = SectionEntry.Value;
 
-			if (Section.ShouldRenderDynamicPathRayTracing())
-			{
-				UMaterialInterface* SectionMat = GetMaterialSlot(Section.MaterialSlot);
-				check(SectionMat);
+	//		if (Section.ShouldRenderDynamicPathRayTracing())
+	//		{
+	//			UMaterialInterface* SectionMat = GetMaterialSlot(Section.MaterialSlot);
+	//			check(SectionMat);
 
-				FRayTracingGeometry* SectionRayTracingGeometry = &Section.Buffers->RayTracingGeometry;
+	//			FRayTracingGeometry* SectionRayTracingGeometry = &Section.Buffers->RayTracingGeometry;
 
-				if (SectionRayTracingGeometry->RayTracingGeometryRHI->IsValid())
-				{
-					check(SectionRayTracingGeometry->Initializer.TotalPrimitiveCount > 0);
-					check(SectionRayTracingGeometry->Initializer.IndexBuffer.IsValid());
+	//			if (SectionRayTracingGeometry->RayTracingGeometryRHI->IsValid())
+	//			{
+	//				check(SectionRayTracingGeometry->Initializer.TotalPrimitiveCount > 0);
+	//				check(SectionRayTracingGeometry->Initializer.IndexBuffer.IsValid());
 
-					FRayTracingInstance RayTracingInstance;
-					RayTracingInstance.Geometry = SectionRayTracingGeometry;
-					RayTracingInstance.InstanceTransforms.Add(GetLocalToWorld());
+	//				FRayTracingInstance RayTracingInstance;
+	//				RayTracingInstance.Geometry = SectionRayTracingGeometry;
+	//				RayTracingInstance.InstanceTransforms.Add(GetLocalToWorld());
 
-					FMeshBatch MeshBatch;
-					CreateMeshBatch(MeshBatch, Section, LODIndex, SectionId, SectionMat, nullptr, true);
+	//				FMeshBatch MeshBatch;
+	//				CreateMeshBatch(MeshBatch, Section, LODIndex, SectionId, SectionMat, nullptr, true);
 
 
-					MeshBatch.CastRayTracedShadow = IsShadowCast(Context.ReferenceView);
+	//				MeshBatch.CastRayTracedShadow = IsShadowCast(Context.ReferenceView);
 
 
-					RayTracingInstance.Materials.Add(MeshBatch);
-					RayTracingInstance.BuildInstanceMaskAndFlags();
-					OutRayTracingInstances.Add(RayTracingInstance);
-				}
-			}
-		}
-	}
+	//				RayTracingInstance.Materials.Add(MeshBatch);
+	//				RayTracingInstance.BuildInstanceMaskAndFlags();
+	//				OutRayTracingInstances.Add(RayTracingInstance);
+	//			}
+	//		}
+	//	}
+	//}
```

Comment:
This whole raytracing function is commented out; need to revert and make appropriate upgrades.